src/import-v1/index.ts

```diff
@@ -0,0 +1,375 @@
+import * as crypto from 'crypto';
+import * as fs from 'fs';
+import * as stream from 'stream';
+import * as util from 'util';
+import * as readline from 'readline';
+import * as path from 'path';
+
+import { DbBnsName, DbBnsNamespace, DbBnsSubdomain } from '../datastore/common';
+import { PgDataStore } from '../datastore/postgres-store';
+import { asyncBatchIterate, asyncIterableToGenerator, logError, logger } from '../helpers';
+
+import { PoolClient } from 'pg';
+
+const IMPORT_FILES = [
+  'chainstate.txt',
+  'name_zonefiles.txt',
+  'subdomains.csv',
+  'subdomain_zonefiles.txt',
+];
+
+const finished = util.promisify(stream.finished);
+const pipeline = util.promisify(stream.pipeline);
+const access = util.promisify(fs.access);
+const readFile = util.promisify(fs.readFile);
+
+const SUBDOMAIN_BATCH_SIZE = 2000;
+
+export class LineReaderStream extends stream.Duplex {
```

Comment:
[`split2`](https://www.npmjs.com/package/split2) package can be used instead, it has less overhead